#Name of classe
std_msgs/String class_name

#Prediction confidence
float32 confidence

# 2D bounding box surrounding the object.
BoundingBox2D bbox

